app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -13,6 +13,10 @@ import AcuantContext from '../context/acuant';
  * @param {(nextValue: any) => void} onChangeCallback Callback to trigger on change.
  */
 export function defineObservableProperty(object, property, onChangeCallback) {
+  if (Object.hasOwn(object, property)) {
```

Comment:
It seems like this should throw?

Reply:
That is, if I call `defineObservableProperty` a second time and it doesn't actually add my callback, it should throw an Error telling me so.

Reply:
OBE